### ISSUE TITLE
What's the best accuracy of  mobilefacenet  on lfw, cfp_fp, agedb_30?

### ISSUE THREAD

**Issue by Wisgon** [deepinsight/insightface#206]
I have only two GPU of 1070ti, so my command is `CUDA_VISIBLE_DEVICES='0,1' python -u train_softmax.py --network y1 --loss-type 4 --margin-s 128 --margin-m 0.5 --per-batch-size 182 --emb-size 128 --ckpt 2 --data-dir ../datasets/faces_ms1m_112x112 --wd 0.00004 --fc7-wd-mult 10.0 --prefix ../mobilenet/model-mobilefacenet-182`
My final accuracy is lfw: 0.98967,     agedb_30: 0.911,     cfp_fp: 0.87471.  And the speed of training is about 900sample/sec.
Is the accuracy normal or too low?

**Comment by Wisgon** [comment]
Can you provide the Pretrained Model of  mobilefacenet ? I only have two GPU of 1070ti, I can't make the 'per_batch_size' bigger, I think the 'per_batch_size' is the main reason that my accuraray is too low.

**Comment by nttstar** [comment]
#214
